lib/child_process.js

```diff
@@ -140,9 +140,21 @@ exports.exec = function(command /*, options, callback*/) {
                           opts.callback);
 };
 
-Object.defineProperty(exports.exec, customPromisifyArgs,
-                      { value: ['stdout', 'stderr'], enumerable: false });
-
+Object.defineProperty(exports.exec, util.promisify.custom, {
+  enumerable: false, value: function(...args) {
```

Comment:
I think we prefer breaking after the `,`, and if you want, you can use the `value(...args) {` shorthand notation